api/src/services/assets.ts

```diff
@@ -40,6 +40,7 @@ export class AssetsService {
 		this.accountability = options.accountability || null;
 		this.filesService = new FilesService({ ...options, accountability: null });
 		this.authorizationService = new AuthorizationService(options);
+		this.filesService = new FilesService({ ...options, accountability: undefined });
```

Comment:
```suggestion
```

Why are we setting `this.filesService` twice here? Did you mean to update line 41 or create a new FilesService under a different property name 🤔

Reply:
Wrong merge conflict fix :)